modules/local/gffread.nf

```diff
@@ -0,0 +1,87 @@
+process GFFREAD {
+    tag "$meta.id"
+    label 'process_single'
+    container = 'ecoflowucl/gffread_python:python-3.11.9_Linux_x86_64_perl-5.36.0'
```

Comment:
Consider using Seqera containers for this. Also please add a conda environment.yml

Reply:
We want to add the nf-core module for this to the pipeline. So won't do this yet. 
The reason we couldn't use the nfcore gffread version was becasue the output breaks the AGAT longest module, so temporarily we are using some custom modules from prior pipelines that we know work, for now